Validation/HLTrigger/src/HLTGenResHistColl.cc

```diff
@@ -0,0 +1,303 @@
+#include "Validation/HLTrigger/interface/HLTGenResHistColl.h"
+
+namespace {
+  // function to get the trigger objects of a specific collection
+  std::pair<trigger::size_type, trigger::size_type> getTrigObjIndicesOfCollection(
+      const trigger::TriggerEvent& triggerEvent, const std::string& collectionName, const std::string& hltProcessName) {
+    trigger::size_type begin = 0;
+    trigger::size_type end = 0;
+    for (trigger::size_type collNr = 0; collNr < triggerEvent.sizeCollections(); ++collNr) {
+      //std::cout <<collNr<<" collection label "<<triggerEvent.collectionTag(collNr).label()<<" "<<triggerEvent.collectionKey(collNr)<<std::endl;
```

Comment:
remove commented out line of code if not needed

Reply:
done